### PR TITLE
Indirect - Data Analysis - Normalize by lowest temperature in ElasticWindowMultiple

### LINKED ISSUE #23002: Indirect - Data Analysis - ElasticWindowMultiple not normalizing by lowest temperature
The ElasticWindowMultiple algorithm is not currently normalizing by the lowest temperature.

### PR DESCRIPTION
**Pull request by brandonhewer**
**Report to:** Sanghamitra

In the ElasticWindowMultiple algorithm, each spectrum should be normalized by the y-value corresponding to the lowest temperature -- this is currently not the case and as a result the output of this algorithm is incorrect.

The ElasticWindowMultiple algorithm already sorts the x-axis -- as such the y-value value corresponding to the lowest temperature is always the first y-value in the spectrum.

This PR modifies the call to normalize_by_index in ElasticWindowMultiple -- simply passing 0.

**To test:**
1. Navigate to Interfaces > Indirect > Data Analysis > Elwin.
2. Provide all of the supplied files as input.
3. Tick the option ‘Normalise to lowest temperature’
4. Click 'Run'.
5. The output `*_elt` workspace should be equivalent to transposing the `*_eq` workspace, sorting the x-axis and then normalizing each column by the first column (as such the first column should be filled with 1s).

**Test Files:**
[elwin_test_data.zip](https://github.com/mantidproject/mantid/files/2209371/elwin_test_data.zip)

(https://github.com/mantidproject/mantid/files/2206818/irs26176_graphite002_red.zip)

Fixes #23002. 

Does this update require release notes?
- [x] Yes
- [ ] No

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by martyngigg** [comment]
The test file doesn't seem to contain any sample logs. Is there another I can use?

**Comment by martyngigg** [comment]
I just get:
"Log file for run irs26173 not found
No sample units found for run: irs26173"
in the sample log when I run it.

File: Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ElasticWindowMultiple.py [hunk 1]
```diff
@@ -210,7 +208,7 @@ def axis_value(index):
                 elt_workspace = CloneWorkspace(InputWorkspace=elf_workspace, OutputWorkspace="__cloned",
                                                StoreInADS=False, EnableLogging=False)
 
-            _normalize_by_index(elt_workspace, np.argmin(sample_param))
+            _normalize_by_index(elt_workspace, 0)
```
Comment (martyngigg) [inline Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ElasticWindowMultiple.py]: Is there a reason that this doesn't require a test to be updated.

**Review by martyngigg — CHANGES_REQUESTED** [review]
I think this could also do with some release notes.

**Comment by keeeto** [comment]
I don't see an `_elt` file - does this mean `_elf`?

**Comment by SpencerHowells** [comment]
@keeeto Did you tick the option ‘Normalise to lowest temerature’?

**Comment by keeeto** [comment]
@SpencerHowells , I did. Thanks for adding that to the instructions. It still does not produce the `_elt` workspace. This seems to be related to Martyn's point about not finding sample logs.

**Comment by keeeto** [comment]
@SpencerHowells - we're having difficulty finding a good sample dataset to test this on. Do you have any suggestions?

**Comment by SpencerHowells** [comment]
@keeeto Try this data it has sample logs
[elwin_test_data.zip](https://github.com/mantidproject/mantid/files/2209357/elwin_test_data.zip)

**Comment by keeeto** [comment]
Thanks @SpencerHowells

**Review by keeeto — APPROVED** [review]
According to the updated instructions and the data provided by @SpencerHowells this now passes the technical tests.

File: Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ElasticWindowMultiple.py (hunk 1 above)
Comment (brandonhewer) [reply Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ElasticWindowMultiple.py]: Currently there are no tests for the ElasticWindowMultiple algorithm,

Comment (martyngigg) [reply Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ElasticWindowMultiple.py]: Ok. Could you make an issue for that?

**Review by martyngigg — APPROVED** [review]

**Pull request merged**